packages/babel-plugin-i18n-calypso/package.json

```diff
@@ -23,7 +23,6 @@
 	"license": "GPL-2.0-or-later",
 	"scripts": {
 		"clean": "npx rimraf dist",
-		"prepublish": "npm run clean",
-		"prepare": "transpile"
+		"prepublish": "npm run clean"
 	}
```

Comment:
We might not need the entire `scripts` section:
The `prepare` script is meant to transpile a package's source, and puts the transpiled code into `dist`.
The other scripts are directly related to that: `clean` removes the `dist/` dir, `prepublish` calls `clean`. But with no `dist/` dir, we won't need those.

Reply:
Removed it in https://github.com/Automattic/wp-calypso/pull/40758/commits/f17e3d7dd6382b787f8acc089872b1f9c6a2b192.